pkg/apiserver/storage/vpc_inventory.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2023 Antrea Authors
```

Comment:
Put this file under pkg/controllers/inventory/store

Reply:
moved